.github/workflows/build_docker_image.yml

```diff
@@ -12,9 +12,8 @@ jobs:
     timeout-minutes: 20
     steps:
       - name: Clone Tree
-        uses: actions/checkout@v1
+        uses: actions/checkout@v2
         with:
-          fetch-depth: 1
```

Comment:
Why the removal?

Reply:
checkout v2 is fetch-depth 1 by default.